audio-worklet/src/bitcrusher-audio-worklet.js

```diff
@@ -39,7 +39,7 @@ registerProcessor('bitcrusher-audio-worklet',
   constructor(options) {
     super(options);
 
-    // Index and previousSample defined as globals to handle block transitions.
+    // Index_ and previousSample_ are globals to handle block transitions.
```

Comment:
Nit: The variable is `index_` and not `Index_`.  I think this is one reason people do `|index_|`.